fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientAddressRepository.java

```diff
@@ -27,8 +27,8 @@
 public interface ClientAddressRepository extends JpaRepository<ClientAddress, Long>, JpaSpecificationExecutor<ClientAddress> {
 
     @Query("SELECT clientAddress FROM ClientAddress clientAddress WHERE clientAddress.client.id = :clientId AND clientAddress.addressType = :addressType AND clientAddress.isActive = :isActive ")
-    ClientAddress findByClientIdAndAddressTypeAndIsActive(@Param("clientId") final long clientId, final CodeValue addressType,
-            @Param("isActive") final boolean isActive);
+    ClientAddress findByClientIdAndAddressTypeAndIsActive(@Param("clientId") final long clientId,
+            @Param("addressType") final CodeValue addressType, @Param("isActive") final boolean isActive);
```

Comment:
hah, nice catch - looks like this was wrong (missing) in #1110 ?!  I missed it! @kiran-vsm FYI...